scss/6-components/_targeted-content.scss

```diff
@@ -50,16 +50,21 @@
       width: 100%;
       padding: $cads-spacing-3 $cads-spacing-4;
       text-align: left;
+      border: 1px solid $cads-language__link-colour;
 
       @include cads-media-breakpoint-down(sm) {
         align-items: baseline;
       }
 
-      @include cads-interactive-element();
```

Comment:
I get that this mixin doesn't do exactly what you want it to, but the focus styles in this mixin are the same as the ones you've added.  Just wondering if there is a way we can tie the two together using extend or another mixin perhaps?

Am a bit wary of having independent instances of these styles in mixins - it gives the impression we can just change the mixin to update interactive styles everywhere, which won't be the case here.

Maybe this isn't a real problem, what do you think?

Reply:
Yeah I did wonder about that, the targeted content felt like a reasonable balance as it already has to do a bunch of "resetting" because of the way that toggle works. I do agree though, maybe we need a lower level mixin for the focus styles. I'll have a look.